samples/19.customization-plain-ui/README.md

```diff
@@ -0,0 +1,53 @@
+# Customizing Web Chat with a plain HTML UI
+
+This sample shows how to use Web Chat without its canned UI component.
+
+When we design Web Chat, we think deeply about customizations. To make customizations great, we design our component in layers. Developers can go into the full experience of Web Chat by using all of our layers as-is. Or they could build their own UI from ground up with our business logic. Or they can opt into our UIs but replacing just a handful of components.
+
+Below is the explanation of different layers of Web Chat.
+
+- [botframework-directlinejs](https://github.com/Microsoft/BotFramework-DirectLineJS) should be eventful and stateless
```

Comment:
should talk about how it's an API to the Bot Framework protocol.

Reply:
Added more details on `botframework-directlinejs`.